oak/server/rust/oak_loader/src/main.rs

```diff
@@ -40,6 +44,8 @@ pub struct Opt {
     private_key: Option<String>,
     #[structopt(long, help = "Path to the PEM-encoded certificate chain")]
     cert_chain: Option<String>,
+    #[structopt(long, help = "Metrics server port number")]
```

Comment:
Could you please add a description that the default port is `3030`?
And also use `#[structopt(default_value = 3030)]`